cql2pgjson/src/test/java/org/folio/cql2pgjson/CQL2PgJSONTest.java

```diff
@@ -942,6 +942,43 @@ public void basicFT(String testcase)
     logger.fine("basicFT: " + testcase + " OK ");
   }
 
+  /**
+   * Isolated fulltext search tests for special characters.
+   *
+   * The use of a single equal operator '=' results in a fulltext search.
+   * Additional characters are tested to ensure that they do not cause any problems.
+   *.
+   * This uses www, xxx, yyy, and zzz to explicitly avoid potential stop-words.
+   *
+   * @param testcase conditions expected to pass.
+   */
+  @Test
+  @Parameters({
+    "text='text' # a; b; c; d; e; f",
+    "text='text without www' # a",
```

Comment:
An CQL search term that contains a space must be enclosed in double quotes: https://www.loc.gov/standards/sru/cql/spec.html
This is different from PostgreSQL where single quotes are used.